src/core/qgsrenderedcalculatedresults.h

```diff
@@ -0,0 +1,88 @@
+/***************************************************************************
+    qgsrenderedcalculatedresults.h
+    ----------------
+    copyright            : (C) 2024 by Jean Felder
+    email                : jean dot felder at oslandia dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSRENDEREDCALCULATEDRESULTS_H
+#define QGSRENDEREDCALCULATEDRESULTS_H
+
+#include "qgis_core.h"
+#include "qgis_sip.h"
+#include "qgsrendereditemdetails.h"
+
+#include <limits>
+
+
+/**
+ * \ingroup core
+ * \brief Contains computed information about rendered features.
+ * \since QGIS 3.42
+ */
+class CORE_EXPORT QgsRenderedCalculatedResults: public QgsRenderedItemDetails
```

Comment:
I'm not 100% on the name "QgsRenderedCalculatedResults". Maybe "QgsRenderedLayerStatistics" or "QgsComputedRenderStatistics" would work better?

Reply:
Changed it to `QgsRenderedLayerStatistics`.